packages/nuxt/src/app/composables/fetch.ts

```diff
@@ -1,30 +1,53 @@
-import type { FetchError, FetchOptions } from 'ofetch'
```

Comment:
Would you be able to revert the code style changes in this file?

(And also, we need to investigate why CI lint step isn't catching this 🤔)

Reply:
Argh, sorry about this. I thought I'd turned off the formatter 🤦 Yes, i'll sort it 👍